terraform/modules/fargate-task-definition/outputs.tf

```diff
@@ -21,3 +21,11 @@ output "container_name" {
   description = "Name of the container"
   value       = var.container_name
 }
+
+output "log_group" {
+  description = "Name of the log group for OS"
```

Comment:
it's not really the name, it's multiple attributes

Reply:
touché